v2/pkgs/tools.go

```diff
@@ -222,12 +223,36 @@ func (t *Tools) install(ctx context.Context, path, url, checksum string) (*tools
 		return nil, err
 	}
 
-	err = extract.Archive(ctx, &buffer, t.folder, rename(path))
+	tempPath := paths.TempDir()
```

Comment:
Should we `defer rempPath.RemoveAll()` too?

Reply:
I don't think so, `paths.TempDir()` simply returns the default path to the user's temporary files.